### ISSUE TITLE
Spoiler Log Issues

### ISSUE THREAD

**Issue by BlueViper85** [Dessyreqt/alttprandomizer#307]
Eudaimonistic missed the heart Container at Bombos during the play through of seed C8705895 ; however, the stats showed 160/160 chests and 49/49 other locations (totaling 209/209).

https://www.twitch.tv/eudaimonistic/v/115114239?t=03h41m22s

Shortly after you'll notice 24/24 heart pieces were obtained, but 10/11 heart containers.  Bombos had a heart container.

As expeceted the locations from the spoiler log of v7 add up to 209 (if you count that Magic Bat isn't listed in the v7 spoiler log) suggesting that something possibly caused the location count to increase erroneously (or perhaps 209 isn't the right number after all?)

**Comment by tim-burget** [comment]
I think the game counts the Magic Bat as a location, but the 209 figure came from somebody forgetting to count the bat when they were determining how many locations there are.

**Comment by BlueViper85** [comment]
Without the magic bat the spoiler log comes to 208 locations, with the bat you get the 209 that the stats show. This includes the bombos tablet, so that suggests that either another location is missing and it should be 210 total (or possibly higher) or something accidentally counted more than once.

**Comment by tim-burget** [comment]
I completed a run where it said I had checked 210 out of 209 locations.

**Comment by BlueViper85** [comment]
Perhaps there's another location that the stats include that spoiler log doesn't have. Thanks!

**Comment by tim-burget** [comment]
Is the game counting Hyrule Castle Escape's Big Key as a location?

**Comment by tim-burget** [comment]
It's either that, or an overworld location.

**Comment by BlueViper85** [comment]
Assuming it's a matter of a missing location. But there are other options. Trying to avoid specific speculation.  The person who can check and fix is aware and will check and remediate whenever he gets to it. Ultimately a minor issue though.

**Comment by BlueViper85** [comment]
Fruitbats just ran a Randomizer and had 5/4 chests. He did throw the sword in twice. So it either counts as 2 locations for the sword (counting the silver arrows) or the sword can be counted repeatedly

**Comment by ChristosOwen** [comment]
From Issue #311:

I completed a run where I got all the Heart Containers, but the ending stats said I missed one of them (seed C4724332).

**Comment by tim-burget** [comment]
I figured out what the 210th location is: It's the Silver Arrows.

**Comment by BlueViper85** [comment]
I've had a similar thought about it being at the Fat Fairy location, and I still suspect it might but I did some testing here.

I beat the game without getting the fat fairy sword or silver arrows and finished the game with 207/209 locations.  I loaded a save state right before Ganon and then went to get silver arrows and the fair fairy sword and finished with 209/209.

So I went back to the save state again, got the silver arrows and fat fairy sword and threw the sword in repeatedly as well as the arrows and then beat it a third time.  This time I got 209/209 locations still (admittedly unexpectedly) but I had 27/27 Y Items and 6/4 swords.  So clearly the fat fairy is counting as an additional location if you throw in at least the sword more than once, but doesn't seem to explain the 210/209.
